### PR TITLE
Add new rule file_permissions_sudo

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

Add new rule file_permissions_sudo

#### Rationale:
To cover ANSSI R38

### CODE REVIEW

File: linux_os/guide/system/software/sudo/file_permissions_sudo/rule.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+description: |-
+{{{ describe_file_permissions("/usr/bin/sudo", "4750") | indent(4) }}}
+
+rationale: |-
+    The sudoers program should only be usable by people who have the correct permissions.
+
+identifiers:
+    cce@rhel7: CCE-86949-5
+    cce@rhel8: CCE-86950-3
+    cce@rhel9: CCE-86951-1
```
Comment (jan-cerny) [inline linux_os/guide/system/software/sudo/file_permissions_sudo/rule.yml:14]: Please remove these CCEs from `shared/references/cce-redhat-avail.txt`.

**Review by jan-cerny — CHANGES_REQUESTED** [review]

**Comment by jan-cerny** [comment]
@Mab879 Testing farm fail is legit, it fail in the rule file_permissions_sudo in the ANSSI profile. The actual permissions are of `/usr/bin/sudo` are 4111.

**Comment by Mab879** [comment]
> @Mab879 Testing farm fail is legit, it fail in the rule file_permissions_sudo in the ANSSI profile. The actual permissions are of `/usr/bin/sudo` are 4111.

So I misread the permissions in ANSSI, they are looking for 4110, but that might not be possible.

File: linux_os/guide/system/software/sudo/file_permissions_sudo/rule.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+severity: medium
+
+platform: package[sudo]
+
+template:
+    name: "file_permissions"
+    vars:
+        filepath: "/usr/bin/sudo"
+        filemode: '4110'
```
Comment (jan-cerny) [inline linux_os/guide/system/software/sudo/file_permissions_sudo/rule.yml:24]: CI still fails.

So is it 4110 or 4111?

The HTML report from the testing farm https://artifacts.dev.testing-farm.io/ae6aeb0b-5999-4a1d-ad23-2291c000eb50/work-ansible-anssivlxuhmw6/tests/fmf-plans/ansible-anssi/execute/data/guest/default-0/Sanity/ansible-machine-hardening/anssi_bp28_high-1/data/anssi_bp28_high.html indicates that 4111 .

On my machine:

```
jcerny@fedora:~$ stat -c %a /usr/bin/sudo
4111
```

But I haven't investigated it further.

File: linux_os/guide/system/software/sudo/file_permissions_sudo/rule.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+documentation_complete: true
+
+title: 'Ensure That the sudo Binary Has the Correct Permissions'
+
+description: |-
+{{{ describe_file_permissions("/usr/bin/sudo", "4750") | indent(4) }}}
```
Comment (jan-cerny) [inline linux_os/guide/system/software/sudo/file_permissions_sudo/rule.yml:6]: the value of the permissions will have to be the same as in the template section

**Comment by Mab879** [comment]
Moving to 4111 as 4110 doesn't seem possible.

**Pull request merged**